Instanssi/main2019/static/main2019/js/radio.js

```diff
@@ -64,10 +67,72 @@ $(function() {
                     console.log('Ei saatu kalenterin tietoja.');
                     return;
                 }
-                parseCal(data.items);
+                if (typeof data !== 'undefined' && typeof data.items !== 'undefined') {
+                    parseCal(data.items);
+                }
+            },
+            complete: function() {
+                setTimeout(calRequest, 300000);     // 5 mins
+            }
+        });
+    }
+
+    function parseSheet(params) {
+        var oldval = unescape($('#radiostream_nytsoi').html());
```

Comment:
Is unescape() necessary here?

Reply:
jQuery text property will escape special HTML characters and API returns utf8 charset.
It is necessary when I do comparison here: (line 109 radio.json):
```
if (updateStr !== oldval) {
            $('#radiostream_nytsoi').html(updateStr);
}
```
Tbh this comparison is not yet necessary. Does not affect user experience..